### ISSUE TITLE
Goldpage does not work on windows

### ISSUE THREAD

**Issue by estyrke** [reframejs/goldpage#21]
When running goldpage build (or goldpage dev) on windows, an internal error occurs (unfortunately I don't have a windows box to try on now, but the error is very terse anyway). It seems related to the assumption that paths are separated by forward slashes, which is not the case on windows.

**Comment by brillout** [comment]
That's possible. I don't have a windows box right now but I'll have a look at it later.

**Comment by brillout** [comment]
Goldpage doesn't support Windows for now. Comment this ticket if you want windows support.

**Comment by littleninja** [comment]
I'm doing a bit of research and discovered Goldpage, love its focus on do-one-thing-do-it-well! Knowing my team works primarily on Windows (I'm on MacOS), we might give Goldpage a spin with WSL but this would be a non-starter for us. 

Assuming this issue may one day be closed as "Won't Fix", it'd be great if there was a note in the README with platform support.

 > It seems related to the assumption that paths are separated by forward slashes, which is not the case on windows.

This tickles my curiosity but I'm not on Windows. I wonder if `path.join()` or `cross-env` ([now in maintenance](https://github.com/kentcdodds/cross-env/issues/257), alternatives include `env-cmd` or `@naholyr/cross-env`) could help.

**Comment by brillout** [comment]
Hi @littleninja !

I will have another look and tentatively write a patch.

**Comment by brillout** [comment]
I'll write a patch tentivally next week. In the meantime, WSL should work.

**Comment by brillout** [comment]
@littleninja Still interested? I've capacity today to write the patch.

**Comment by littleninja** [comment]
Sorry for the delay, I hope my comment didn't come across in a way that shifted priorities. There's no urgency on my part. 😔

**Comment by brillout** [comment]
No worries.

>  There's no urgency

You may wanna wait on https://github.com/reframejs/goldpage/issues/24 then. It's going to be lovely.

If you don't mind me asking; what exactly are your plans with Goldpage? We can discuss in depth about whether Goldpage is/will be a good fit for you, and it's insightful for me to talk about my users' use cases.

**Comment by brillout** [comment]
@littleninja FYI https://github.com/brillout/vite-plugin-ssr

**Comment by brillout** [comment]
@littleninja https://github.com/brillout/vite-plugin-ssr is pretty much done and I published a beta. I'd be curious to know what you think.
